kotlin/injivcrenderer/src/main/java/io/mosip/injivcrenderer/InjiVcRenderer.kt

```diff
@@ -0,0 +1,53 @@
+package io.mosip.injivcrenderer
+import java.text.SimpleDateFormat
+import java.util.*
+
+class InjiVcRenderer: InjiVcRendererInterface {
+
+
+    fun getValueFromData(key: String, data: Map<String, Any?>): Any? {
+        val keys = key.split("/")
+        var value: Any? = data
+        for (k in keys) {
+            if (value is Map<*, *>) {
+                value = (value as Map<String, Any?>)[k]
+            } else {
+                return null
+            }
+        }
+        return value
+    }
+
+    fun formatDate(dateString: String): String {
```

Comment:
duplicate method found in DateUtils.kt as well.

Reply:
Done